api/src/membership/membership.py

```diff
@@ -198,7 +198,7 @@ def get_access_summary(member_id: int):
     from multiaccessy.accessy import accessy_session
 
     if accessy_session is None:
-        return PreconditionFailed(f"Accessy is not configured.")
+        return dict(in_org=False, pending_invite_count=0, access_permission_group_names=[])
```

Comment:
I think it's good practice to give the return value a name so that its intent becomes clearer:

```suggestion
    dummy_access_summary = dict(in_org=False, pending_invite_count=0, access_permission_group_names=[])
    if accessy_session is None:
        return dummy_access_summary
```